app/services/cfe_civil/components/employments.rb

```diff
@@ -4,9 +4,11 @@ class Employments < BaseDataBlock
       delegate :employments, to: :legal_aid_application
 
       def call
-        return {} unless employment_and_payments?
-
-        { employment_income: employment_income_payload }.to_json
+        if employment_and_payments?
+          { employment_income: employment_income_payload }.to_json
+        else
+          {}.to_json
+        end
```

Comment:
I toyed with the idea of 
```ruby
      def call
        if employment_and_payments?
          { employment_income: employment_income_payload }
        else
          {}
        end
      end.to_json

```
but I couldn't handle the nausea it caused! 😄 

I also tried 
```ruby
      def call
        result = if employment_and_payments?
                   { employment_income: employment_income_payload }
                 else
                   {}
                 end
        result.to_json
      end
```
but it just seemed a bit noisy